src/CodeMirror.svelte

```diff
@@ -135,6 +126,9 @@
 	}
 
 	onMount(() => {
+		if (!codemirror_promise) {
+			codemirror_promise = import('./codemirror.js');
+		}
```

Comment:
Something weird happened here with this change - `_CodeMirror` is never getting set anywhere anymore. This dynamic loading of `codemirror.js` all looks like it could be aimplified and tidied a bit, too.

Reply:
Oh, I just copied your change from https://github.com/sveltejs/svelte/pull/6811#issuecomment-935939154

Maybe we don't need both the `_CodeMirror` and `CodeMirror` variables?

Reply:
I think it would look more natural if we do the dynamic import in the async IIFE below, maybe like:

```js
(async () => {
  let mod = await import('./codemirror.js');
  CodeMirror = mod.default;
})()
```

I don't see why we need to use variables in `context="module"`.

Reply:
I've updated this a bit, so that it's cleaner now